LICENSE

```diff
@@ -706,3 +706,247 @@
+        origin of the Work and reproducing the content of the NOTICE file.
+
+    7. Disclaimer of Warranty. Unless required by applicable law or
+        agreed to in writing, Licensor provides the Work (and each
+        Contributor provides its Contributions) on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+        implied, including, without limitation, any warranties or conditions
+        of TITLE, NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A
+        PARTICULAR PURPOSE. You are solely responsible for determining the
+        appropriateness of using or redistributing the Work and assume any
+        risks associated with Your exercise of permissions under this License.
+
+    8. Limitation of Liability. In no event and under no legal theory,
+        whether in tort (including negligence), contract, or otherwise,
+        unless required by applicable law (such as deliberate and grossly
+        negligent acts) or agreed to in writing, shall any Contributor be
+        liable to You for damages, including any direct, indirect, special,
+        incidental, or consequential damages of any character arising as a
+        result of this License or out of the use or inability to use the
+        Work (including but not limited to damages for loss of goodwill,
+        work stoppage, computer failure or malfunction, or any and all
+        other commercial damages or losses), even if such Contributor
+        has been advised of the possibility of such damages.
+
+    9. Accepting Warranty or Additional Liability. While redistributing
+        the Work or Derivative Works thereof, You may choose to offer,
+        and charge a fee for, acceptance of support, warranty, indemnity,
+        or other liability obligations and/or rights consistent with this
+        License. However, in accepting such obligations, You may act only
+        on Your own behalf and on Your sole responsibility, not on behalf
+        of any other Contributor, and only if You agree to indemnify,
+        defend, and hold each Contributor harmless for any liability
+        incurred by, or claims asserted against, such Contributor by reason
+        of your accepting any such warranty or additional liability.
+
+    END OF TERMS AND CONDITIONS
+
+    APPENDIX: How to apply the Apache License to your work.
+
+        To apply the Apache License to your work, attach the following
+        boilerplate notice, with the fields enclosed by brackets "[]"
+        replaced with your own identifying information. (Don't include
+        the brackets!)  The text should be enclosed in the appropriate
+        comment syntax for the file format. We also recommend that a
+        file or class name and description of purpose be included on the
+        same "printed page" as the copyright notice for easier
+        identification within third-party archives.
+
+    Copyright [yyyy] [name of copyright owner]
```

Comment:
Should be filled in...

Reply:
I think this is fine. There isn't one particular copyright to put there because this is just a reproduction of all of the third-party licenses that each have their own respective copyrights

Reply:
Hmm, IDK about that. Might actually have to reproduce each one with the proper copyright info displayed. Did you get a green light on this from some Apache document or counsel?

Reply:
Well, the licensing information is typically checked when we submit a new release for a vote on the Incubating General list. This is the format we had it in for 1.4 (when it finally passed) and there were not any comments against the format.

The copyrights for the various source dependencies are also in this file. The appendix just contains the full text of the license for reference and because the licenses usually contain the stipulation that releases contain the full text of the license. So, there isn't any information missing about the copyrights and licensing here.

Reply:
I think you can delete that section anyways - it's boilerplate language to be included in the headers of files. So it both places it could be deleted.
https://github.com/apache/incubator-mxnet/blob/f71d87de5bd69e7e57552de4c52fbdd97e2ca5ff/LICENSE#L178-L203
https://github.com/apache/incubator-mxnet/blob/f71d87de5bd69e7e57552de4c52fbdd97e2ca5ff/LICENSE#L892-L917
But, I'd just refer to the Apache License at the beginning of the doc, rather than including it again.